rotkehlchen/db/evmtx.py

```diff
@@ -686,20 +706,20 @@ def get_or_create_genesis_transaction(
                 )
         return tx
 
-    def _form_evm_transaction_dbquery(self, query: str, bindings: list[Any], has_premium: bool) -> tuple[str, list[tuple]]:  # noqa: E501
+    def _form_evm_transaction_dbquery(self, query: str, bindings: list[Any], has_premium: bool) -> tuple[str, list]:  # noqa: E501
         """Return query and bindings for the evm_transaction database table"""
+        base_select = 'SELECT evm_transactions.tx_hash, evm_transactions.chain_id, timestamp, block_number, from_address, to_address, value, gas, gas_price, gas_used, input_data, evm_transactions.nonce, identifier, auth.nonce AS auth_nonce, auth.delegated_address'  # noqa: E501
+        join_clause = 'LEFT JOIN evm_transactions_authorization auth ON evm_transactions.tx_hash = auth.tx_hash AND evm_transactions.chain_id = auth.chain_id'  # noqa: E501
```

Comment:
what is the change in this function? I don't see it clearly

Reply:
i added a join to include the authorizations for each transaction. broke it into multiple lines to make it more readable.